cmd/store.go

```diff
@@ -23,52 +24,46 @@ func (s *store) Get(IP string) (string, error) {
 	if role, ok := s.rolesByIP[IP]; ok {
 		return role, nil
 	}
+	log.Warnf("Using fallback role for IP %s", IP)
```

Comment:
I don't think we need to move this log, we don't know if we are going to fallback to the default role yet.